### PR TITLE
Hide maker toggle when it's a level without maker enabled

### PR DESCRIPTION
**Pull request by caleybrock**
Don't show the maker toggle in design or code mode for levels without maker enabled

Level without maker
screen shot 2017-11-20 at 12 02 11 pm

Level with maker
screen shot 2017-11-20 at 12 04 19 pm

Project level
screen shot 2017-11-20 at 12 02 18 pm

### CODE REVIEW

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -604,7 +604,8 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugSlider: showDebugConsole,
-    showDebugWatch: !!config.level.isProjectLevel || config.level.showDebugWatch
+    showDebugWatch: !!config.level.isProjectLevel || config.level.showDebugWatch,
+    showMakerToggle: !!config.level.isProjectLevel || config.level.makerlabEnabled,
```
Comment (caleybrock) [inline apps/src/applab/applab.js:608]: This is the actual logic of when we hide/show the toggle. 
Always show for projects or if we are on a level with maker enabled.

File: apps/src/applab/DesignModeHeaders.jsx [hunk 2]
```diff
@@ -91,7 +92,7 @@ export default class DesignModeHeaders extends React.Component {
 
     const hasFocus = !this.props.isRunning;
 
-    const settingsCog = <SettingsCog isRunning={this.props.isRunning} runModeIndicators/>;
+    const settingsCog = <SettingsCog isRunning={this.props.isRunning} showMakerToggle={this.props.showMakerToggle} runModeIndicators />;
```
Comment (Bjvanminnen) [inline apps/src/applab/DesignModeHeaders.jsx]: nit: i would make these be multiple lines

**Comment by caleybrock** [comment]
@Bjvanminnen added a test - please take a look

**Comment by caleybrock** [comment]
cc @islemaster - who originally made this component

**Pull request merged**